ext/drbbdup/drbbdup.c

```diff
@@ -558,7 +577,11 @@ drbbdup_is_at_end(instr_t *check_instr)
 
     if (instr_is_cti(check_instr)) {
         instr_t *next_instr = instr_get_next(check_instr);
-        return drbbdup_is_at_label(next_instr, DRBBDUP_LABEL_START);
+        return drbbdup_is_at_label(next_instr, DRBBDUP_LABEL_START) ||
+            /* There may be an emulation endpoint label in between. */
+            (next_instr != NULL && instr_get_next(next_instr) != NULL &&
+             instr_is_label(next_instr) &&
```

Comment:
Is there a way to add to the condition that the label is actually an emulation endpoint label, and not just check it is a generic label?

Reply:
Yes we can use drmgr_is_emulation_{start,end}.  Will put that in.  But first there's a crash in PR #5393 -- first making sure that is not related to more that needs to be done here before pushing changes.